ev_charge/lib/database/db_manager.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:drift/drift.dart';
+import 'package:flutter/foundation.dart';
+
+import 'database.dart';
+
+class DbManager {
+  static Future<AppDatabase> initialize({
+    bool debugMode = false,
+    bool clearDB = false,
+  }) async {
+    final db = AppDatabase();
+
+    if (clearDB) {
+      await _resetDatabase(db);
+    }
+
+    if (debugMode) {
+      if (kDebugMode) {
+        print('[Debug] Seeding database...');
+      }
+      await _seedDatabase(db);
+    }
+    return db;
+  }
+
+  static Future<void> _seedDatabase(AppDatabase db) async {
+    // Avoid duplicate seeding — only insert if empty
+    final existingEVModels = await db.select(db.eVCarModels).get();
+    final existingUserEVs = await db.select(db.userEVs).get();
+    final existingConstraints = await db.select(db.constraints).get();
+    final existingSchedules = await db.select(db.schedules).get();
+
+    if (existingEVModels.isEmpty) {
+      // Seed EVCarModels
+      if (kDebugMode) {
+        print('[Seed] EVCarModels seeding.');
+      }
+      await db.batch((batch) {
+        batch.insertAll(db.eVCarModels, [
+          EVCarModelsCompanion.insert(
+            modelName: 'Model S',
+            modelYear: const Value(2020),
+            batteryCapacity: 100.0,
+            maxChargingPower: 250.0,
+          ),
+          EVCarModelsCompanion.insert(
+            modelName: 'Model 3',
+            modelYear: const Value(2022),
```

Comment:
```suggestion
            modelYear: const Value(2020),
            batteryCapacity: 100.0,
            maxChargingPower: 250.0,
          ),
          EVCarModelsCompanion.insert(
            modelName: 'Model 3',
            modelYear: const Value(2022),
```

De her 'const Value(x)' er også forbundet til 
```diff
- late final modelYear = integer().nullable()();
+ late final modelYear = integer()();
```